packages/bot-skeleton/src/services/api/active-symbols.js

```diff
@@ -143,11 +162,18 @@ export default class ActiveSymbols {
             const { submarkets } = market_obj;
 
             Object.keys(submarkets).forEach(submarket_name => {
-                submarket_options.push([submarkets[submarket_name].display_name, submarket_name]);
+                const { display_name } = submarkets[submarket_name];
+                const submarket_display_name =
+                    display_name + (this.isSubmarketClosed(submarket_name) ? ` ${localize('(Closed)')}` : '');
```

Comment:
```suggestion
               `${display_name}${this.isSubmarketClosed(submarket_name)? localize('(Closed)') : ''}`;
```

Reply:
IMO the template literal makes it less clear. Won't change.